src/feature/PuppetStatusBarFeature.ts

```diff
@@ -64,15 +71,39 @@ export class PuppetStatusBar {
     this.statusBarItem.tooltip = toolTip; // TODO: killme (new Date()).getUTCDate().toString() + "\nNewline\nWee!";
   }
 
-  public static showConnectionMenu() {
+  public showConnectionMenu() {
     var menuItems: PuppetConnectionMenuItem[] = [];
 
     menuItems.push(
       new PuppetConnectionMenuItem(
         "Show Puppet Session Logs",
         () => { vscode.commands.executeCommand(PuppetCommandStrings.PuppetShowConnectionLogsCommandId); }),
     );
-  
+
+    if (this.config.ruby.pdkPuppetVersions !== undefined && this.config.ruby.pdkPuppetVersions.length > 0) {
```

Comment:
`this.config.ruby.pdkPuppetVersions` is 0 on the mac, but greater than 0 on windows

Reply:
Found the problem and fixed it.  Adding tests now.

Reply:
Fixed!!!